src/test/regress/spec/isolation_drop_vs_all.spec

```diff
@@ -2,30 +2,37 @@
 // How we organize this isolation test spec, is explained at README.md file in this directory.
 //
 
-// create range distributed table to test behavior of DROP in concurrent operations
+// create distributed table to test behavior of DROP in concurrent operations
 setup
 {
 	SELECT citus_internal.replace_isolation_tester_func();
 	SELECT citus_internal.refresh_isolation_tester_prepared_statement();
 
 	SET citus.shard_replication_factor TO 1;
-	CREATE TABLE drop_hash(id integer, data text);
-	SELECT create_distributed_table('drop_hash', 'id');
+	CREATE SCHEMA drop_tests
+		CREATE TABLE drop_hash(id integer, data text);
+	SELECT create_distributed_table('drop_tests.drop_hash', 'id');
+
+	CREATE SCHEMA drop_tests_2
+		CREATE TABLE drop_hash_2(id integer, data text);
+	SELECT create_distributed_table('drop_tests_2.drop_hash_2', 'id');
 }
 
 // drop distributed table
```

Comment:
the `//` comments breaks isolation tests on MacOs, can you please update to `#` ? Thanks!

Reply:
We do not allow `#` comments in spec files. The reason is that we now use C preprocessor, and it expects valid directives after this character.

Any `spec` file that has `$` comments would break our CI checks that are introduced in https://github.com/citusdata/citus/pull/5471

If this prevents running isolation tests on MacOS, then we should come up with a better solution that works for all linux distros (that possibly have outdated C preprocessors in their toolkit) and MacOS.

However, to have passing tests, I am inclined to keep the files as is now.